scripts/cross-compile

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+DOCKER_USER="node"
+if [[ "$TRAVIS" == "true" ]]; then DOCKER_USER="travis"; fi
```

Comment:
@ralphtheninja Can you explain the users? I'm guessing the working directory on the host is owned by `travis`, and docker must use the same user? Can't we change permissions instead?

Reply:
I'm open to suggestions. If we don't do anything about the user (e.g. set `USER` in the `Dockerfile`) the container will run as root, thus producing binaries owned by root. Which leaves us with two options (as I see it):

1. either we do something related to user, i.e. run the container with the correct user
2. keep running the container as root, but copy out the binaries from the container, which is done in `prebuildify-cross`

Reply:
Thinking about it some more, I think I prefer 2. since the docker image doesn't need any specific users. This needs some tweaking to the `cross-compile` script as well. Also need to look into how `prebuildify-cross` does this with output folders etc.

Reply:
> Because it runs as user with uid 1000 and the jenkins user is 2000.

*travis user? 😉 

Alright, I'm fine with both the current solution and the alternative, copying out the binaries. The latter would also remove the `npm i` problem (https://github.com/Level/leveldown/pull/587#discussion_r253047869), because if we copy out the binaries then we don't have to mount any volume.

Reply:
Yes, travis user. I  don't know where jenkins came from. _Jenkins! Get ut of my head!_

Reply:
It's not a perfect solution, but it works. Lets tweak it as we go?